picard/album.py

```diff
@@ -552,6 +553,31 @@ def switch_release_version(self, mbid):
             self.tagger.albums[mbid] = self
             self.load(priority=True, refresh=True)
 
+    def update_metadata_images(self):
+        new_images = []
+        orig_images = []
+
+        def process_file_images(file):
+            for image in file.metadata.images:
+                if image not in new_images:
+                    new_images.append(image)
+            try:
+                for image in file.orig_metadata.images:
+                    if image not in orig_images:
+                        orig_images.append(image)
+            except AttributeError:
+                pass
+
+        for track in self.tracks:
+            process_file_images(track)
+            for file in list(track.linked_files):
+                process_file_images(file)
+        for file in list(self.unmatched_files.files):
```

Comment:
I had a reason for those, but I forgot... so ok, I'll change it (and the other case)